src/main/java/Parser.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Represents a parser that helps to make sense of the user commands.
+ */
+public class Parser {
+    /**
+     * Reads the user input and returns the corresponding command according to the command
+     * given by the user.
+     *
+     * @param input The user input into the program.
+     * @return The corresponding command for the user input.
+     * @throws DukeException
+     */
+    public static Command parse(String input) throws DukeException {
+        input = input.toLowerCase();
+        String[] inputWords = input.split(" ");
+
+        if (inputWords.length == 0) {
+            throw new DukeException("Sorry, I don't understand what you are saying! D=\n"
+                    + "Type \"help\" to view the list of commands you can use!");
+        }
```

Comment:
As for me, I find the multiple "if"s not as readable as "if-else if" blocks, but even if the logic permits, it is still totally up to you whether to change it or not.